modules/local/bbmap_phix.nf

```diff
@@ -40,4 +40,21 @@ process BBMAP_PHIX {
           bbduk: \$(bbversion.sh | grep -v "Duplicate cpuset")
         END_VERSIONS
         """
+
+
+        stub:
+        def prefix = task.ext.prefix ?: "${meta.id}"
+
+        """
+        touch ${prefix}_clean_R1.fastq.gz ${prefix}_clean_R2.fastq.gz
+        touch ${prefix}_noclean_R1.fastq.gz ${prefix}_noclean_R2.fastq.gz
+
+        touch ${prefix}.stats_phix.txt
+        touch ${prefix}.bbmap_phix.log
+
+        cat <<-END_VERSIONS > versions.yml
+        "${task.process}":
+          bbduk: \$(bbversion.sh | grep -v "Duplicate cpuset")
+        END_VERSIONS
+        """
 }
```

Comment:
The `stub` section, only really needs to generate the files mentioned in the `output` (via `touch` ) - but I've added the version file generation as a proper test in case we want to use the actual conda/docker environment in future with stubs.